cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/CgmesExport.java

```diff
@@ -27,22 +38,50 @@
 
     @Override
     public void export(Network network, Properties params, DataSource ds) {
-
-        // Right now the network must contain the original CgmesModel
-        // In the future it should be possible to export to CGMES
-        // directly from an IIDM Network,
-        // without the need for the original CgmesModel
+        Objects.requireNonNull(network);
         CgmesModelExtension ext = network.getExtension(CgmesModelExtension.class);
-        if (ext == null) {
-            throw new CgmesModelException("No extension for CGMES model found in Network");
+        if (params != null && Boolean.valueOf(params.getProperty("cgmes.export.usingOnlyNetwork"))) {
+            if (ext != null) {
+                CgmesModel cgmesSource = ext.getCgmesModel();
+                if (cgmesSource != null) {
+                    throw new CgmesModelException("CGMES model should not be available as Network extension");
+                }
+            }
+            exportUsingOnlyNetwork(network, ds);
+        } else {
+            if (ext == null) {
+                throw new CgmesModelException("CGMES model is required and not found in Network extension");
+            }
+            exportUsingOriginalCgmesModel(network, ds, ext);
         }
-        CgmesUpdate cgmesUpdate = ext.getCgmesUpdate();
+    }
 
+    private void exportUsingOnlyNetwork(Network network, DataSource ds) {
+        // At this point only SSH, SV can be exported when relying only in Network data
+        // (minimum amount of CGMES references are expected as aliases/properties/extensions)
+        String baseName = network.hasProperty("baseName") ? network.getProperty("baseName") : network.getNameOrId();
+        String filenameSv = baseName + "_SV.xml";
+        String filenameSsh = baseName + "_SSH.xml";
+        CgmesExportContext context = new CgmesExportContext(network);
+        try (OutputStream os = ds.newOutputStream(filenameSv, false)) {
+            XMLStreamWriter writer = XmlUtil.initializeWriter(true, "    ", os);
+            StateVariablesExport.write(network, writer, context);
+        } catch (IOException | XMLStreamException x) {
+            throw new PowsyblException("Exporting to CGMES using only Network");
+        }
```

Comment:
I don't really agree with throwing a `PowsyblException` in any case. You should do:
```java
catch (IOException e) {
   throw new UncheckedIOException(e);
} catch (XMLStreamException e) {
    throw new UncheckedXmlStreamException(e);
}
```

You can write this only once at the end of the method (not twice)

Reply:
Done. Defined a single try-with-resources block for export SV and SSH export